resolwe/flow/managers/celery.py

```diff
@@ -7,25 +7,38 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import logging
 import sys
 
+from resolwe.utils import BraceMessage as __
+
 from ..tasks import celery_run
 from .base import BaseManager
 
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
 try:
     import celery  # pylint: disable=unused-import
 except ImportError:
-    print('Please install Celery using `pip install celery`', file=sys.stderr)
+    print("Please install Celery using `pip install celery`", file=sys.stderr)
```

Comment:
Please, use `logger.error` here and replace `` ` ``s with ordinary quotes (`'`).

Reply:
I left this as it was pre-refactor, apart from the outer single quotes.